docs/deployment/1_launch_subnet.md

```diff
@@ -5,58 +5,70 @@ sidebar_position: 1
-
-    1. How many machines will you use to deploy subnet?
-    2. How many subnet nodes will you deploy in total?
-    3. IP address of the main machine
-    4. Parentchain wallet with funds
-
-  In this setup the main machine (machine1) will host all the subnet services(relayer, stats, frontend) while the subnet miner nodes will be spread out among all machines.
-  
-  The IP address of the main machine is needed for subnet connectivity, this is the IP that is known to all other machines, could be private or public (preferrably private)
-
-  Once generated, the commands to startup the subnet is also provided as a generated `commands.txt` file.
-
-  The deployment is docker based, the main deployment file is `docker-compose.yml`. The `docker-compose.env` is the injection point to all configs. Then, ENVs for the services and subnet nodes are in `*.env` files. Other files include `genesis.json` file to initialize subnet chain, `deployment.json` to deploy the checkpoint smartcontract, and `keys.json` the keypairs for subnet nodes + grandmaster node. Again, these files will be generated by the Subnet Deployment Generator (SDG).
-
 ## Requirements
-  - OS: Linux. Only Linux is supported for full deployment. 
+  - OS: Linux. Only Linux is supported for full deployment.
 
-  - OS: Mac is only supported for single machine testing environment. Specify MacOS with 'OS=mac' in 'docker.env' file. Please also refer [common issues](./2_debug_guide.md#common-issues).
+  - OS: Mac is only supported for single machine testing environment.
   
   - docker, docker compose V2. For manual installation of docker compose V2 please refer to: https://docs.docker.com/compose/install/linux/
   
-## Steps
-  1. Create a `docker.env` file with parameters similar to [`docker.env.example`](https://github.com/XinFinOrg/XinFin-Node/blob/master/subnet/deployment-generator/script/docker.env.example). Detailed parameters explanation [here](./3_configs_explanation.md).
+  - Recommended Hardware (per single Subnet node):
+    - CPU: 2 Core
+    - Memory: 4 GB
+
+  - Web3 wallet with funds. For testing we have faucets provided:
+    - https://faucet.apothem.network/ 
+    - https://faucet.blocksscan.io/
 
-  2. Pull the `generator.sh` script from the generator Github repo
+## Video Walkthrough
+<iframe width="720" height="400" src="https://www.youtube.com/embed/m-sPbMrB8ow" frameborder="0" allow="fullscreen"></iframe>
+
+## Generate Subnet Configs With UI
+
+  1. Pull `generator.sh` script from the generator Github repo
   ```
-  curl -O https://raw.githubusercontent.com/XinFinOrg/XinFin-Node/master/subnet/deployment-generator/script/generate.sh
+  curl -O https://raw.githubusercontent.com/XinFinOrg/XinFin-Node/generator-ui-concept/subnet/deployment-generator/scripts/generate.sh
```

Comment:
master or you will change it later?

Reply:
this is convenient for testing will change before merge